tests/authentication_tests/test_interactive_authentication.py

```diff
@@ -302,3 +306,8 @@ def test_ui_auth_with_whitespace(self):
 
 # Class decorators don't exist on Python2.5
 InteractiveTests = skip_if_pyside_missing(InteractiveTests)
+
+def tearDownModule():
+    from tank_test.tank_test_base import tearDownModule as baseTearDownModule
```

Comment:
why do we need this `tearDownModule` that isn't defined anywhere else in our tests? What is it that we are cleaning up here? Please add some more comments to clarify!